server/service/hosts.go

```diff
@@ -148,6 +148,7 @@ func (svc *Service) ListHosts(ctx context.Context, opt fleet.HostListOptions) ([
 	}
 	filter := fleet.TeamFilter{User: vc.User, IncludeObserver: true}
 
+	// TODO(Sarah): Are we missing any other filters here?
```

Comment:
Looks like this story is premium only: https://github.com/fleetdm/fleet/issues/10936

So the bootstrap filter should be applied only if premium AFAICT. Technically, I don't think it would cause any issues to leave it be, as it wouldn't find anything in any status IIUC, so it would always return an empty list for all statuses, but probably best to clear it so that it doesn't apply (and mention it in the docs that this is premium-only).